.github/workflows/ci.yml

```diff
@@ -298,6 +298,7 @@ jobs:
     timeout-minutes: 60
     permissions:
       pull-requests: write
+      checks: write
```

Comment:
Why do we need this permission?

Reply:
To be honest, I am not sure if we actually need it. We shouldn't.

The issue is that summary entries and comments do not appear in forks-originated PRs; they do appear in PRs from this repo. 

So my initial assumption is that: 

* Either `github.event_name` in case of `pull_request_target` is not `'pull_request'` (so I changed the `if` to `if: ${{ success() && github.event_name != 'push' }}`) 
* Or this permission is needed (because it is also present in this workflow https://github.com/elastic/cloudbeat/blob/2451b25449ded5bef249d134eede8148bce16efe/.github/workflows/publish-test-results.yml#L24)

Do you think we should first merge the first only, and if it does not work, then we should try the `check: write` permission?

Reply:
Let's attempt to merge without it (checks: write), and if necessary, we can add this permission later.